starlight/templates/profile_views/profile.html

```diff
@@ -0,0 +1,32 @@
+{% extends "base.html" %}
+
+{% load staticfiles %}
+
+{% load i18n %}
+
+{% block content %}
+
+<h1 class="ui black image header">
+    <div class="header">
+        {% trans "Your Competencies: " %}
+    </div>
+</h1>
+<br>
+<h1 class="ui black image header">
+    <div class="content">
+        {% for competency in employee.competencies.all %}
+        {{competency}}
```

Comment:
The information for these competencies should be shown in a table format, like so:

Skill | Interest | Experience | Actions
----- | ---------- | -------------- | ---------
JavaScript | 3. Competent | 1. Less than two years | _edit button_